cwms-data-api/src/main/java/cwms/cda/api/BasinController.java

```diff
@@ -61,13 +65,13 @@ private Timer.Context markAndTime(String subject) {
                             + "their parameters."),
             },
             responses = {
-                    @OpenApiResponse(status = "200",
+                    @OpenApiResponse(status = STATUS_200,
```

Comment:
In hindsight, Javalin really should've made these integers so the existing constants could be used.